### DISCUSSION TITLE
Export From Dax Studio to SQL ,with automatique incremental refresh

### DISCUSSION THREAD

**Discussion by JijaGallery** [DaxStudio/DaxStudio#1058] [Q&A]
Hi ,

I find nice solution to export the data to SQL table . But I couldn't find how I can to set incremental refresh automatically once a week. 
can u help me how it is possible.

**Comment by dgosbell** [comment]
You cannot set an automatic incremental export using DAX Studio. You would have to write your own script or extract of some sort to query the data you needed and load it into an SQL table.

**Comment by fudanzakura** [comment]
Sql comes with import function, or it can be imported with software similar to navicat
